src/components/m-button/mixins/_button.scss

```diff
@@ -290,8 +299,24 @@
       background: $color-button-bg--motion--hover;
     }
   }
+
+  &:disabled {
```

Comment:
Ok i said lets increase specificty instead of using important.. but i guess as we already used a lot of importants we can also here.